assets/app/index.html

```diff
@@ -65,7 +65,9 @@
               <li class="divider"></li>
               -->
               <li>
-                <a href="logout">Log out</a>
+                <!-- Stop event propagation to avoid a runtime error when the
+                     dropdown tries to close itself as we're navigating away. -->
+                <a href="logout" ng-click="$event.stopPropagation()">Log out</a>
```

Comment:
ng-cloak here definitely reduces flicker in the banner on load.

On Sunday, October 11, 2015, Jordan Liggitt <notifications@github.com>
wrote:

> In assets/app/index.html
> <https://github.com/openshift/origin/pull/4869#discussion_r41717735>:
>
> > @@ -65,7 +65,9 @@
> >                <li class="divider"></li>
> >                -->
> >                <li>
> > -                <a href="logout">Log out</a>
> > +                <!-- Stop event propagation to avoid a runtime error when the
> > +                     dropdown tries to close itself as we're navigating away. -->
> > +                <a href="logout" ng-click="$event.stopPropagation()">Log out</a>
>
> not sure ng-cloak will help with things outside the main controller div
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4869/files#r41717735>.
>